client/styles/components/_nav.scss

```diff
@@ -3,26 +3,35 @@
 .nav {
   height: #{math.div(42, $base-font-size)}rem;
   display: flex;
+  width: 100%;
   flex-direction: row;
   justify-content: space-between;
 
-  @include themify() {
-    border-bottom: 1px dashed map-get($theme-map, 'nav-border-color');
-  }
-
   & button {
     padding: 0;
   }
 }
 
+.nav__header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 100%;
```

Comment:
I think line 19 might need to be removed, since it seems to be affecting the layout for the user pages and examples!